### ISSUE TITLE
Enable scroll wheel to scroll right side panel

### ISSUE THREAD

**Issue by sloretz** [ignitionrobotics/ign-gazebo#366]
Currently the scroll wheel can only scroll the right panel if it is exactly over the scroll bar. It would be easier to scroll the panel if the scroll wheel could be used over the content being scrolled - just like scrolling in a web browser or text editor.

![Peek 2020-09-23 09-37](https://user-images.githubusercontent.com/4175662/94042392-6035c800-fd80-11ea-96ad-2b0e3fba8253.gif)

**Comment by chapulina** [comment]
Yeah the tricky thing is that each individual plugin may also be scrollable, so when you scroll on top of the plugins, they're swallowing the scroll events.

I think the ideal behaviour is for the plugins to only swallow the scroll if they can be scrolled. Otherwise, they let the scroll event bubble up to the panel.

Like this text box on GitHub:

![githubscroll](https://user-images.githubusercontent.com/5751272/94045853-cae90280-fd84-11ea-8206-73d0ec7944cd.gif)


Related to https://github.com/ignitionrobotics/ign-gui/pull/97
